specifyweb/specify/datamodel.py

```diff
@@ -1591,7 +1593,8 @@
             Relationship(name='rightSideRels', type='one-to-many',required=False, relatedModelName='CollectionRelationship', otherSideName='rightSide', dependent=True),
             Relationship(name='treatmentEvents', type='one-to-many',required=False, relatedModelName='TreatmentEvent', otherSideName='collectionObject', dependent=True),
             Relationship(name='visibilitySetBy', type='many-to-one',required=False, relatedModelName='SpecifyUser', column='VisibilitySetByID'),
-            Relationship(name='voucherRelationships', type='one-to-many',required=False, relatedModelName='VoucherRelationship', otherSideName='collectionObject', dependent=True)
+            Relationship(name='voucherRelationships', type='one-to-many',required=False, relatedModelName='VoucherRelationship', otherSideName='collectionObject', dependent=True),
+            Relationship(name='cotype', type='many-to-one', required=True, relatedModelName='CollectionObjectType', column='CoTypeID'),
```

Comment:
what is cotype refers to?

Reply:
collectionobjecttype, I thought abbreviating it would be better, but I can change it if it's not clear.

Reply:
can we capitalize the C and the O?

Reply:
is "coType" ok?  None of the datamodel relationships start with a capital letter.  In the database schema it's "CoType".